### DISCUSSION TITLE
International/localizer.

### DISCUSSION THREAD

**Discussion by NathanX-S** [open-toontown/open-toontown#70] [Q&A]
In an attempt to re-create my "remove product name/international logic" PR, I realized something. It might be a bad idea to remove \all\ international logic, especially the custom positioning for certain languages and such. Do we plan to support multiple localizers?
CC: @LittleToonCat

**Comment by opentoontown** [comment]
It's not something that's planned for any time in the foreseeable future, however we should be mindful to not remove anything that allows for us (or someone forking the project) to support it in the future.
